sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ModelsRepositoryClientOptions.cs

```diff
@@ -14,7 +14,7 @@ public class ModelsRepositoryClientOptions : ClientOptions
         internal const ServiceVersion LatestVersion = ServiceVersion.V2021_02_11;
 
         /// <summary>
-        /// The versions of Azure IoT Model Repository by this client
+        /// The versions of Azure IoT Models Repository supported by this client.
```

Comment:
add a service in there :D 
`/// The versions of Azure IoT Models Repository service supported by this client.`

Reply:
Good call!!